session 03/src/App.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
```

Comment:
have a new line between library imports and local file imports.

Reply:
Added new line